columnar/src/columnar/writer/serializer.rs

```diff
@@ -102,7 +107,7 @@ mod tests {
         let mut buffer: Vec<u8> = b"somegarbage".to_vec();
         prepare_key(b"root\0child", ColumnType::Str, &mut buffer);
         assert_eq!(buffer.len(), 12);
-        assert_eq!(&buffer[..10], b"root\0child");
```

Comment:
which sep do we use for fields in columnar? is it \1?

Reply:
yes `\u{1}`